src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it. To avoid overflow or accidental scroll bar. (CSS checklist)